### PR TITLE
Feature/86cv5erqr service card

### PR DESCRIPTION
**Pull request by NikkiShao**
Service cards that have a brief description of the service. Currently displaying mock data.

The cards change as screen size changes, on mobile devices, the artist's name and image are omitted for clarity.

Buttons should take users to the specific service's detail page (not completed yet I believe so a dummy page was created). Clicking on an artist's profile should take the user to the artist's page.

### CODE REVIEW

File: behind-the-veil-siteroot/imports/ui/components/card/ServiceCard.jsx
Comment (nethbtju) [inline behind-the-veil-siteroot/imports/ui/components/card/ServiceCard.jsx:1]: Probably should fix the ui for this like we discussed

**Review by nethbtju — CHANGES_REQUESTED** [review]
UI needs changing as discussed, ticket needs to be refined

File: behind-the-veil-siteroot/imports/ui/components/card/ServiceCard.jsx [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+                    <div className="small-text text-dark-grey max-h-[4.5rem] max-w-full line-clamp-4 break-all
+                    ">{serviceDesc}</div>
+                </div>
+
+                {/* clickable to navigate to artist profile*/}
+                <div className={"hidden sm:flex flex-col items-center justify-center cursor-pointer min-w-20"}
+                     onClick={() => navigateTo('/artists/' + artistId)}
+                >
+                    <ProfilePhoto artistPhotoData={artistPhotoData}></ProfilePhoto>
+                    <div className="max-h-[4rem] large-text text-our-black line-clamp-2 text-center">{artistName}</div>
```
[the remark this answers is not in the document]
Comment (NikkiShao) [reply behind-the-veil-siteroot/imports/ui/components/card/ServiceCard.jsx]: Artist names length taken into consideration, will be present on the artists page now

File: behind-the-veil-siteroot/imports/ui/components/card/ServiceCard.jsx
Comment (NikkiShao) [reply behind-the-veil-siteroot/imports/ui/components/card/ServiceCard.jsx:1]: Fixed as discussed

**Review by nethbtju — APPROVED** [review]
All issues fixed

**Pull request merged**